### PR TITLE
op-node/derive: linter fix

### PR DESCRIPTION
**Pull request by KailMillor**
linter fix, replace `fmt.Errorf` with `errors.New` when pure content

### CODE REVIEW

File: op-node/rollup/derive/l1_block_info.go [hunk 1]
```diff
@@ -210,19 +210,19 @@ func (info *L1BlockInfo) unmarshalBinaryEcotone(data []byte) error {
 		return err
 	}
 	if err := binary.Read(r, binary.BigEndian, &info.BaseFeeScalar); err != nil {
-		return fmt.Errorf("invalid ecotone l1 block info format")
```
Comment (tynes) [inline op-node/rollup/derive/l1_block_info.go:213]: This is a good change but would be better if there was a sentinel error that was returned in all cases

Comment (KailMillor) [reply op-node/rollup/derive/l1_block_info.go:213]: Nice advice!

**Comment by ajsutton** [comment]
Looks like you need to run gofmt: https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/59929/workflows/f31cbaad-5b6b-4613-9d2f-35a3096a4470/jobs/2521657

**Comment by KailMillor** [comment]
> Looks like you need to run gofmt: https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/59929/workflows/f31cbaad-5b6b-4613-9d2f-35a3096a4470/jobs/2521657

Fixed PLZ check Again @ajsutton

**Review by ajsutton — APPROVED** [review]

**Pull request merged**